yarn.lock

```diff
@@ -17316,16 +17422,17 @@ __metadata:
   languageName: node
   linkType: hard
 
-"vue-tsc@npm:^0.34.8":
-  version: 0.34.17
-  resolution: "vue-tsc@npm:0.34.17"
+"vue-tsc@npm:^0.40.13":
+  version: 0.40.13
+  resolution: "vue-tsc@npm:0.40.13"
   dependencies:
-    "@volar/vue-typescript": 0.34.17
+    "@volar/vue-language-core": 0.40.13
+    "@volar/vue-typescript": 0.40.13
   peerDependencies:
     typescript: "*"
   bin:
     vue-tsc: bin/vue-tsc.js
-  checksum: 0c5addb114b6e4d20e131ebdc95806fc7b723f2301735b929f3e9e70491787f13b0318b25b75af50e32c6639754b62c62182483a1eda0fa63448436f3e82a505
+  checksum: 00e5cb424f1ab1a198489400b165a1d3968f85bf2747fe1cc8d8ac7161752a407d68e703f0132a547eaa8b236d1454627b92771cf268a968e5ac191b18de9690
```

Comment:
this one I know but the other change I dont catch why if the dependencies are the same